server/mongodb/models/Chapter.ts

```diff
@@ -1,35 +1,30 @@
 import mongoose, { Schema } from "mongoose";
 import AddressSchema from "server/mongodb/models/embedded/Address";
-import { Chapter, NonprofitStage, ProjectType } from "src/utils/types";
+import { Chapter, MaintenanceType } from "src/utils/types";
 
 const ChapterSchema = new Schema<Chapter>({
   name: {
     type: String,
     required: true,
   },
-  email: {
-    type: String,
+  email: String,
+  contact: {
```

Comment:
We will need to populate this field in the future, so make this a reference to user.  Check this out for more info, we will be using it often: https://mongoosejs.com/docs/populate.html